ozaria/site/components/interactive/PageInteractive/draggableOrdering/OrderingSlots.vue

```diff
@@ -0,0 +1,101 @@
+<script>
+  import BaseDraggableSlot from '../BaseDraggableSlot'
+
+  export default {
+    components: {
+      'base-draggable-slot': BaseDraggableSlot
+    },
+
+    props: {
+      draggableGroup: {
+        type: String,
+        required: true
+      },
+
+      numSlots: {
+        type: Number,
+        default: undefined
+      },
+
+      value: {
+        type: Array,
+        default: () => ([])
+      }
+    },
+
+    data () {
+      let slotData = []
+
+      if (typeof this.numSlots !== 'undefined') {
+        slotData = Array(this.numSlots)
+          .fill(undefined)
```

Comment:
Is numSlots only for empty slots?

Reply:
Yes. I believe `numSlots` is used to initialize empty slots that can be filled later.